src/components/calcite-input-message/readme.md

```diff
@@ -9,10 +9,10 @@ Displays a contextual message to a user. Allows the passing of content, links, e
 ### Basic
 
 ```html
-<calcite-label>
+<calcite-label status="“invalid”">
   My great label
-  <calcite-input placeholder="“Enter" your information” status="invalid"></calcite-input>
-  <calcite-input-message active status="invalid"
+  <calcite-input placeholder="“Enter" your information”></calcite-input>
```

Comment:
Is this a typo?

Reply:
Oh, looks like it. 😱 Sneaky fancy quote!

Reply:
lol. This example...

> “Enter" your information”
> That's not going to work out.

:)

Reply:
well you're not wrong... it is invalid after all